ocr/static/ocr/js/rbi_table.js

```diff
@@ -4,61 +1,64 @@
+const $table = $('#table')
+const $remove = $('#remove')
+let selections = []
 
-function getIdSelections() {
-  return $.map($table.bootstrapTable('getSelections'), (row) => row.id);
+function getIdSelections () {
+  return $.map($table.bootstrapTable('getSelections'), (row) => row.id)
 }
 
 /* eslint no-unused-vars:0 */
-function responseHandler(res) {
+function responseHandler (res) {
   $.each(res.rows, (i, row) => {
     /* eslint no-param-reassign:0 */
-    row.state = $.inArray(row.id, selections) !== -1;
-  });
-  return res;
+    row.state = $.inArray(row.id, selections) !== -1
+  })
+  return res
 }
 
-function detailFormatter(index, row) {
-  const html = [];
+function detailFormatter (index, row) {
+  const html = []
   $.each(row, (key, value) => {
-    html.push(`<p><b>${key}:</b> ${value}</p>`);
-  });
-  return html.join('');
+    html.push(`<p><b>${key}:</b> ${value}</p>`)
+  })
+  return html.join('')
 }
 
-function operateFormatter(value, row, index) {
+function operateFormatter (value, row, index) {
   return [
     '<a class="remove" href="javascript:void(0)" title="Remove">',
     '<i class="fas fa-trash"></i>',
-    '</a>',
-  ].join('');
+    '</a>'
+  ].join('')
 }
 
 window.operateEvents = {
 /* eslint func-names:0 */
   'click .remove': function (e, value, row, index) {
     $table.bootstrapTable('remove', {
       field: 'id',
-      values: [row.id],
-    });
-  },
-};
+      values: [row.id]
+    })
+  }
+}
 
-function totalTextFormatter(data) {
-  return 'Total';
+function totalTextFormatter (data) {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=8186851)
```suggestion
function totalTextFormatter(data) {
```